src/components/issues/Card.module.scss

```diff
@@ -11,7 +11,7 @@ $card-text: #78716c;
     transition: 0.2s ease-out;
     display: flex;
     flex-direction: column;
-    margin: 2rem auto;
+    margin: 1rem auto;
```

Comment:
please use even numbers, also why was this changed?

Reply:
Why use Even not Odd? For styling purpose.

Reply:
Can you provide me with the docs for it?